App crashes after closing it

**Describe the bug**
App always crashes after closing it with the back button

**To Reproduce**
1. Open the app
2. Press the back button to exit

**Expected behavior**
The app exists with no further messages

**Screenshots**

<img src="https://user-images.githubusercontent.com/26164108/98460738-8b00a200-21af-11eb-919c-9af5749e9b68.jpg" width="450"/>


**Smartphone:**
 - Device: **Xiaomi Redmi 4X**
 - OS: **MIUI** on Android 7.1.2
 - Version: **11.0.2**

**Additional context**

The generated exception list:

`java.lang.RuntimeException: Unable to destroy activity {to.dev.dev_android/to.dev.dev_android.activities.MainActivity}: java.lang.IllegalArgumentException: Service not registered: to.dev.dev_android.util.AndroidWebViewBridge$connection$1@1ca8a03
 at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:4293)
 at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:4311)
 at android.app.ActivityThread.-wrap6(ActivityThread.java)
 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1586)
 at android.os.Handler.dispatchMessage(Handler.java:102)
 at android.os.Looper.loop(Looper.java:163)
 at android.app.ActivityThread.main(ActivityThread.java:6238)
 at java.lang.reflect.Method.invoke(Native Method)
 at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:933)
 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:823)
Caused by: java.lang.IllegalArgumentException: Service not registered: to.dev.dev_android.util.AndroidWebViewBridge$connection$1@1ca8a03
 at android.app.LoadedApk.forgetServiceDispatcher(LoadedApk.java:1297)
 at android.app.ContextImpl.unbindService(ContextImpl.java:1520)
 at android.content.ContextWrapper.unbindService(ContextWrapper.java:658)
 at to.dev.dev_android.util.AndroidWebViewBridge.terminatePodcast(AndroidWebViewBridge.kt:118)
 at to.dev.dev_android.activities.MainActivity.onDestroy(MainActivity.kt:70)
 at android.app.Activity.performDestroy(Activity.java:7068)
 at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1154)
 at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:4280)`

--- a/app/src/main/java/to/dev/dev_android/util/AndroidWebViewBridge.kt
+++ b/app/src/main/java/to/dev/dev_android/util/AndroidWebViewBridge.kt
@@ -114,10 +114,12 @@ class AndroidWebViewBridge(private val context: Context) {
     fun terminatePodcast() {
         timer?.cancel()
         timer = null
-        audioService?.pause()
-        context.unbindService(connection)
-        audioService = null
-        context.stopService(Intent(context, AudioService::class.java))
+        audioService?.let {
+            it.pause()
+            context.unbindService(connection)
+            context.stopService(Intent(context, AudioService::class.java))
+            audioService = null
+        }
     }
 
     fun podcastTimeUpdate() {

